modules/ossm-config-smcp-kiali.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//* service_mesh/v2x/ossm-reference-kiali.adoc
+:_content-type: REFERENCE
+[id="ossm-smcp-kiali_{context}"]
+= Specifying Kiali configuration in the SMCP
+
+You can configure Kiali under the `addons` section of the `ServiceMeshControlPlane` resource. Kiali is enabled by default. To disable Kiali, set `spec.addons.kiali.enabled` to `false`.
+
+You can specify your Kiali configuration in either of two ways:
+
+* Specify the Kiali configuration in the `ServiceMeshControlPlane` resource under `spec.addons.kiali.install`. There are some limitations with this approach, as the complete list of Kiali configurations is not available in the SMCP.
+
+* Configure and deploy a Kiali instance and specify the name of the Kiali resource as the value for `spec.addons.kiali.name` in the `ServiceMeshControlPlane` resource. You must create the CR in the same namespace as the Service Mesh control plane, for example, `istio-system`. If a Kiali resource matching the value of `name` exists, the control plane will use the existing installation. This approach lets you fully customize your Kiali configuration in the Kiali resource.  Note that with this approach, various fields in the Kiali resource are overwritten by the {ProductShortName} Operator, specifically, the `accessible_namespaces` list, as well as the endpoints for Grafana, Prometheus, and tracing.
+
+
+
+.Example SMCP parameters for Kiali
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  addons:
+    kiali:
+      name: kiali
+      enabled: true
+      install:
+        dashboard:
+          viewOnly: false
+          enableGrafana: true
+          enableTracing: true
+          enablePrometheus: true
+        service:
+          ingress:
+            contextPath: /kiali
+----
+
+.Kiali parameters in the SMCP
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  addons:
+    kiali:
+      name:
+|Name of Kiali custom resource. If a Kiali CR matching the value of `name` exists, the {ProductShortname} Operator will use that CR for the installation. If no Kiali CR exists, the Operator will create one using this `name` and the configuration options specified in the SMCP.
+|string
+|`kiali`
+
+|kiali:
+  enabled:
+|This parameter enables/disables Kiali. Kiali is enabled by default.
+|`true`/`false`
+|`true`
+
+|kiali:
+  install:
+|Install a Kiali resource if the named Kiali resource is not present.
```

Comment:
Shouldn't you say something like, "This `install` section is ignored if `addons.kiali.enabled` is `false`" ??